homeassistant/components/incomfort/water_heater.py

```diff
@@ -43,17 +41,20 @@ async def async_setup_platform(
 class IncomfortWaterHeater(IncomfortEntity, WaterHeaterEntity):
     """Representation of an InComfort/Intouch water_heater device."""
 
-    def __init__(self, client, heater) -> None:
+    def __init__(self, client: InComfortGateway, heater: InComfortHeater) -> None:
         """Initialize the water_heater device."""
         super().__init__()
 
-        self._unique_id = f"{heater.serial_no}"
-        self.entity_id = f"{WATER_HEATER_DOMAIN}.{DOMAIN}"
-        self._name = "Boiler"
-
         self._client = client
         self._heater = heater
 
+        self._attr_unique_id = heater.serial_no
+        self._attr_name = "Boiler"
+        self._attr_should_poll = True
+        self._attr_min_temp = 30.0
+        self._attr_min_temp = 80.0
+        self._attr_temperature_unit = UnitOfTemperature.CELSIUS
```

Comment:
Idem

Reply:
Also corrected the second `_attr_min_temp`, should be `_attr_max_temp` instead.